gui/elements/qt/_layer_buttons.py

```diff
@@ -1,47 +1,43 @@
-from PyQt5.QtCore import Qt, QSize
+from PyQt5.QtCore import Qt
 from PyQt5.QtGui import QIcon
 from PyQt5.QtWidgets import QWidget, QVBoxLayout, QHBoxLayout, QPushButton, QFrame, QCheckBox
-from ._layerList import QtLayerList
 from os.path import dirname, join, realpath
-from numpy import empty
 
 dir_path = dirname(realpath(__file__))
-path_delete = join(dir_path,'icons','delete.png')
-path_add = join(dir_path,'icons','add.png')
-path_off = join(dir_path,'icons','annotation_off.png')
-path_on = join(dir_path,'icons','annotation_on.png')
+path_delete = join(dir_path, '..', '..', 'icons','delete.png')
+path_add = join(dir_path, '..', '..', 'icons','add.png')
+path_off = join(dir_path,'..', '..', 'icons','annotation_off.png')
+path_on = join(dir_path,'..', '..', 'icons','annotation_on.png')
```

Comment:
Here I suggest using the scikit-image approach of defining a `data_dir` once, in say `gui/elements/__init__`, and then using that:

```
from .... import assets_dir

path_delete = join(assets_dir, 'icons', 'delete.png')
```

etc. It might also be a good spot to use pathlib. If `assets_dir` is a Path, then it's

```
path_delete = assets_dir / 'icons' / 'delete.png'
```

Reply:
Ok nice, I will try and look at how scikit-image does this and change it. Thanks for flagging.

Reply:
I have fixed this now, defining an `icons_dir`. I'm still using `join` but could switch to pathlib if desired.